src/components/SubscribeSection/styles.ts

```diff
@@ -0,0 +1,162 @@
+import styled, { css } from "styled-components";
+
+export const StyledSubscribeSection = styled.div<{ platform: string }>`
+  display: flex;
+  gap: 64px;
+  align-items: center;
+  margin: 64px 0px;
+  background-color: var(--background-200);
+  border-radius: 20px;
+  padding: 40px;
+  position: relative;
+  max-width: 1280px;
+
+  ${({ platform }) =>
+    platform !== "web" &&
+    css`
+      flex-direction: column;
+      max-width: 87%;
+      padding: 40px 16px 16px 16px;
+    `}
+
+  & .SBC-content {
+    align-items: flex-start;
+    display: flex;
+    flex: 1;
+    flex-direction: column;
+    flex-grow: 1;
+    gap: 20px;
+    position: relative;
+
+    ${({ platform }) =>
+      platform !== "web" &&
+      css`
+        width: 100%;
+      `}
+  }

```

Comment:
Could you please create and export separate component for "Content" only styles?